src/Commands/ArtCommand.php

```diff
@@ -23,17 +51,37 @@ class ArtCommand extends TerminusCommand
      *
      * @param string $name Artwork name
      *
-     * @usage Displays the list of available artwork.
```

Comment:
Also, rather than removing this line, let's change it to say that it displays a random artwork.

Reply:
There were two `@usage` lines, so I removed this one and left the other.